src/instruction.js

```diff
@@ -0,0 +1,4 @@
+class Instruction {
+
+}
+export default Instruction;
```

Comment:
What went well:

- I think that I have done a good job of following TDD principles, by first writing failing tests, then writing production code to pass the tests.
- With the use of the Instruction class, I was able to loosely couple my code and maintain high cohesiveness. This was done by using the instruction class that took in an airport, and plane object...and only having the relevant methods written in the relevant classes. For example, the instruction class simply calls on the airport's `landPlane()` method. It doesn't handle the landing of the planes itself.